lib/themes/dosomething/paraneue_dosomething/js/campaign/prototype.js

```diff
@@ -25,6 +27,17 @@ function makeKudosRequest($el, id)
 }
 
 /**
+ * Make an API request to Rogue to create/delete the Reaction for that item.
+ *
+ * @param string User's northstar ID.
+ */
+ function makeKudosRequestToRogue($northstarId, $postId)
```

Comment:
this is a placeholder for the next chunk of work to send new reactions to Rogue. Is it ok to keep here?